### PR TITLE
Update README with examples and FAQ

### PR DESCRIPTION
**Pull request by philsturgeon**
Removed some old and slightly confusing examples, and instead focus on the basic CLI usage.

### CODE REVIEW

**Comment by philsturgeon** [comment]
@ww-daniel-mora Hey Daniel, I got you something.

File: README.md [hunk 1]
```diff
@@ -4,128 +4,121 @@
...
 
+#### Server Validation
 
-`configure DNS and routing to point http://x.com to "localhost:3000"`
+The server name (and any base path) are ignored entirely, but if you would like to validate a
+specific server as allowed by the API description you can provide the `__server` query parameter.
 
-`configure DNS and routing to point http://x.com/v1 to "localhost:3001"`
+```bash
+$ http GET http://localhost:4010/a?__server=http://example.com
```
Comment (chris-miaskowski) [inline README.md]: Pretty sure that one isn't supported yet ;)

**Comment by ww-daniel-mora** [comment]
Thanks @philsturgeon

**Comment by philsturgeon** [comment]
@ww-daniel-mora fancy some beta access for v3? phil@stoplight.io and I'll sort it out.

**Pull request merged**